slatedb/src/db.rs

```diff
@@ -199,10 +237,10 @@ impl DbInner {
             .expect("write notifier closed");
 
         // if the write pipeline task exits then this call to rx.await will fail because tx is dropped
-        let current_table = rx.await??;
-
+        // TODO: this can be modified as awaiting the last_durable_seq watermark & fatal error.
+        let mut durable_watch = rx.await??;
```

Comment:
Nit: Mind calling this `durable_watcher` for consistency?

Reply:
fixed in 21412d6